roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -6,8 +6,6 @@ script:
   indexed: on
 
 index:
-  number_of_shards: {{ es_number_of_shards | default ('1') }}
```

Comment:
We should be dropping these... this file should not be in the PR.  was work around for #3770